api-samples/push/background.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code is a modified version of
+// https://codelabs.developers.google.com/codelabs/push-notifications
+
+const applicationServerPublicKey = '<key>';
+
+// Whether the extension will attempt to show a notification or not. Controlled
+// by subscribeUserVisibleOnlyFalse() and subscribeUserVisibleOnlyTrue();
+should_show_notif = true;
+
+// Call this to subscribe the extension to a push notification server with
+// `userVisibleOnly: false` to receive silent messages.
+function subscribeUserVisibleOnlyFalse() {
```

Comment:
Given how similar this is to the function below, can we just have one function with a parameter?